source/standards/logging.html.md.erb

```diff
@@ -1,108 +1,47 @@
 ---
-title: Logging
+title: Store and query logs
 expires: 2018-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-## Short-term querying of logs
+For incident management and debugging you should have a system in place so you can store and query your infrastructure and application logs.
 
-For the purposes of incident management and debugging, it's useful to
-have a system for storing and querying the most recent logs produced
-by your application and infrastructure.
+GDS has a shared account with [Logit](https://logit.io/), offering on-demand shared [ELK (Elasticsearch, Logstash and Kibana)](https://www.elastic.co/elk-stack) stacks as a service. ELK offers suitable short-term storage for your logs so they are both accessible and queryable.
```

Comment:
This should be more decisive. Teams should be moving to the central Logit install by default.

Also, I think that second ELK (ELK offers...) should be Logit for clarity.

Reply:
Agreed